lib/WUI/link_content/prusa_link_api.cpp

```diff
@@ -98,10 +98,11 @@ namespace {
             filename[fname_real_len] = '\0';
             return nullopt;
         } else {
-            return StatusPage(Status::NotFound, parser.status_page_handling(), parser.accepts_json);
+            return StatusPage(Status::NotFound, parser);
         }
     }
 
+    //TODO
```

Comment:
I wanted to change the method so that it also uses the new `StatusPage` constructor, but then I forgot and left the TODO there :sweat_smile: , thanks for the reminder.